### ISSUE TITLE
Alternative to systemd?

### ISSUE THREAD

**Issue by zakiloki0** [termux/proot-distro#240]
### Problem description

I know this tag "bug" is not correct but i didn't find "Question" so...

What is the alternative to start services if we have not Systemd?

Thanks!

### What steps will reproduce the bug?

Systemd

**Comment by zakiloki0** [comment]
> You need to execute service utilities manually, like on Termux itself.
> 
> For example, instead of `systemctl start ssh` you need run `sshd`.

always with "d" in the end?

**Comment by ravener** [comment]
> > You need to execute service utilities manually, like on Termux itself.
> > 
> > For example, instead of `systemctl start ssh` you need run `sshd`.
> 
> always with "d" in the end?
> 
> Edit: I tested with some apps and doesn´t work.

No, the d in sshd stands for daemon (as opposed to the ssh client) and it depends on the program, they may add a d suffix or not, and you just run that program normally.

**Comment by TomJo2000** [comment]
Termux itself does have [`runit`](https://en.wikipedia.org/wiki/Runit) available as a service manager; [you would use the `sv` command](https://wiki.archlinux.org/title/runit#Usage) with that.
The `<program>d` naming scheme is followed by `ssh`, `rsync` and a few other applications, but it isn't a universal standard for running a "Daemon" version of a program.
